src/tests/Equinor.ProCoSys.Completion.WebApi.IntegrationTests/CompletionContextExtension.cs

```
@@ -123,22 +129,57 @@ public static void Seed(this CompletionContext dbContext, IServiceProvider servi
+
+        SeedWorkOrder(
+            dbContext,
+            plant,
+            KnownPlantData.OriginalWorkOrderGuid[plant]);
+
+        SeedWorkOrder(
+            dbContext,
+            plant,
+            KnownPlantData.WorkOrderGuid[plant]);
+        var swcrNo = 10;
+        SeedSWCR(
+            dbContext,
+            plant,
+            KnownPlantData.SWCRGuid[plant],
+            ++swcrNo);
+
+        SeedDocument(
+            dbContext,
+            plant,
+            KnownPlantData.DocumentGuid[plant]);
     }
 
-    private static void EnsureCurrentUserIsSeeded(CompletionContext dbContext, ICurrentUserProvider userProvider)
+    private static void SeedWorkOrder(CompletionContext dbContext, string plant, Guid guid)
     {
-        var personRepository = new PersonRepository(dbContext, userProvider);
-        var seeder = personRepository.GetByGuidAsync(userProvider.GetCurrentUserOid()).Result;
-        if (seeder is null)
-        {
-            SeedCurrentUserAsPerson(dbContext, userProvider);
-        }
+        var workOrder = new WorkOrder(plant, guid, Guid.NewGuid().ToString().Substring(0, WorkOrder.NoLengthMax));
+        var workOrderRepository = new WorkOrderRepository(dbContext);
+        workOrderRepository.Add(workOrder);
+        dbContext.SaveChangesAsync().Wait();
+    }
+
+    private static void SeedSWCR(CompletionContext dbContext, string plant, Guid guid, int no)
+    {
+        var swcr = new SWCR(plant, guid, no);
+        var swcrRepository = new SWCRRepository(dbContext);
+        swcrRepository.Add(swcr);
+        dbContext.SaveChangesAsync().Wait();
+    }
+
+    private static void SeedDocument(CompletionContext dbContext, string plant, Guid guid)
+    {
+        var document = new Document(plant, guid, Guid.NewGuid().ToString());
+        var documentRepository = new DocumentRepository(dbContext);
+        documentRepository.Add(document);
+        dbContext.SaveChangesAsync().Wait();
```

From Brandon Minnicks talk at NDC Porto, I learned we should never use .Wait() but instead use GetAwaiter().GetResult()
Is also recommended by Stephen Cleary: https://blog.stephencleary.com/2014/12/a-tour-of-task-part-6-results.html
It provides an aggregate exception instead of a possible silent failure.